.eslintrc.js

```diff
@@ -25,6 +25,7 @@ module.exports = {
           },
         ],
         '@typescript-eslint/no-floating-promises': 'error',
+        '@typescript-eslint/no-empty-function': 'off',
```

Comment:
This was blocking my builds occasionally and does not specify exactly what the issue might be. Following https://stackoverflow.com/questions/76457373/cannot-read-properties-of-undefined-reading-gettokens-occurred-while-linting.